sources/pciclient/src/private.rs

```diff
@@ -12,6 +13,15 @@ use crate::{
 
 const AMAZON_VENDOR_CODE: &str = "1d0f";
 const EFA_KEYWORD: &str = "efa";
+const NEURON_DEVICES: [&str; 7] = [
+    "7064", // inf1 device id 0
+    "7065", // inf1 device id 1
+    "7066", // inf1 device id 2
+    "7067", // inf1 device id 3
+    "7164", // trn1 device id 0
+    "7264", // inf2 device id 0
+    "7364", // trn2 device id 0
```

Comment:
This is unfortunate that we might have to update this list as new devices are launched. I mostly wanted to call it out that we will need to watch for new devices as they launch to enable support here in the future.